tests/test_benchmarking/test_data/test_simulated_data/test_mock_student_provider.py

```diff
@@ -105,7 +175,7 @@ def test_create_mock_students__student_has_num_values_for_attribute_within_range
     def test_create_mock_students__students_have_correct_friend_and_enemy_count(self):
         for i in [1, 5, 10]:
             students = create_mock_students(
-                number_of_students=i + 1,
+                number_of_students=i * 2 + 1,
```

Comment:
Q: why was this change needed?

Reply:
Cause otherwise in the `i=1` case there aren't enough students in the class to satisfy the friends and enemies req

Reply:
Ah gotcha